packages/jest-reporters/src/__tests__/GitHubActionsReporter.test.ts

```diff
@@ -8,17 +8,25 @@
 import type {
   AggregatedResult,
   AssertionResult,
+  Status,
   Test,
   TestCaseResult,
   TestResult,
 } from '@jest/test-result';
 import type {Config} from '@jest/types';
-import GitHubActionsReporter from '../GitHubActionsReporter';
+import {normalizeIcons} from '../../../../e2e/Utils';
```

Comment:
Can you move this helper to `@jest/test-utils`?

Reply:
I would probably do that in another PR though

Reply:
Would be good to this now. `@jest/test-utils` is an internal package.